ail_vru_uc1_scenario/cdasim_config/cdasim/logback.xml

```diff
@@ -224,8 +226,8 @@
             <logger name="org.eclipse.mosaic.fed.ns3.ambassador.Ns3AmbassadorError" additivity="false" level="INFO">
                 <appender-ref ref="CommunicationDetailsLog"/>
             </logger>
-            
-            <logger name="org.eclipse.mosaic.fed.ns3.ambassador" additivity="false" level="DEBUG">
+
+            <logger name="org.eclipse.mosaic.fed.ns3.ambassador" additivity="false" level="INFO">
```

Comment:
Changing this log level removes are ability to inspect individual NS3 communication latency. @MishkaMN is that fine for verification testing.

Reply:
Technically we don't need it. However, I think it might be worth adding it. Previous DEBUG toggling was logging excessively, but I think that is actually turned off now in the image as I requested from Cheng. So the remaining DEBUG logs look manageable size. So let's turn it on just in case.